.nuspec/maui-blazor.aotprofile.txt

```diff
@@ -6567,6 +6643,62 @@ Methods:
 	void System.Text.Json.WriteStack:EnsurePushCapacity ()
 	void System.Text.Json.WriteStack:Pop (bool)
 	void System.Text.Json.WriteStack:Push ()
+	void System.Text.RegularExpressions.Regex:.cctor ()
+	void System.Text.RegularExpressions.Regex:.ctor (string,System.Globalization.CultureInfo)
```

Comment:
Do we know which commit started using Regex? this in itself might slow down startup some.

Reply:
Ok, yeah if it is only in the `maui-blazor` template, it might be something in ASP.NET.

Maybe we don't worry about this one for now.

Reply:
I'm not really sure on this one.

Maybe this PR created a path that now executes the RegEx here?
https://github.com/dotnet/maui/commit/c14e363fa8bc90ef82532ea795f523156a5d0205

We don't really make use of RegEx much on the Android side of things. 
https://github.com/dotnet/maui/search?q=RegEx

Reply:
It could also be: https://github.com/dotnet/maui/commit/87af742bb9a6ed17cede4e4806348e51c5693cc1

I'll get a speedscope of latest Maui and look for the stack trace.

Reply:
There does seem to be some cost in loading Regex the first time, at:

https://github.com/dotnet/maui/blob/e9852b3cdbe8445737fbcb3124385652c18e6b5f/src/BlazorWebView/src/SharedSource/StaticContentHotReloadManager.cs#L20

![image](https://user-images.githubusercontent.com/840039/163626468-39348f7d-fe11-4a1e-8783-ba2b41b5e2b1.png)

[maui-blazor.zip](https://github.com/dotnet/maui/files/8498033/maui-blazor.zip)

This was kind of a franken build with https://github.com/xamarin/xamarin-android/pull/6931 and dotnet/maui 6.0.300-rc.2.5465. I had to turn off AOT, because I was getting a crash.